packages/contracts/test/UnirepApp.test.ts

```diff
@@ -35,8 +36,13 @@ describe('Unirep App', function () {
 
     // epoch length
     const epochLength = 300
-    // generate random user id
-    const id = new Identity()
+    // generate random hash user id
+    const hash = crypto.createHash('sha3-224')
+    const hashUserId = `0x${hash
+        .update(new Identity().toString())
+        .digest('hex')}`
+    const id = new Identity(hashUserId)
+    console.log('Random hashed user id: ', hashUserId)
 
     it('deployment', async function () {
         const [deployer] = await ethers.getSigners()
```

Comment:
There is no tests here, could we put this to the "before" hook?

Reply:
Hi, @ryanycw @iverJisty, what would you think my modification in this part (it's also in #39 ): 
```typescript
describe('Unirep App', function() => {
  let unirep
  let app
  let hashUserId
  let id
  
  // epoch length
  const epochLength = 300
  
  before(async () => {
      // generate random hash user id
      const hash = crypto.createHash('sha3-224')
      hashUserId = `0x${hash
          .update(new Identity().toString())
          .digest('hex')
      }`
      id = new Identity(hashUserId)
  })
...
```
just want to check if this is good.

Reply:
Do you think we can also move the deploy part in before? Since there is no checks in deploy.